.gitignore

```diff
@@ -3,3 +3,4 @@
 .tool-versions
 .DS_Store
 .idea
+Gemfile.lock
```

Comment:
The lockfile should be committed; ensuring anyone that contributes is running the same versions of all the rubygems.

Reply:
Thanks, put it back